.github/workflows/release-wheels.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - uses: actions/setup-python@v4
       name: Install Python
       with:
-        python-version: '3.10'
+        python-version: '3.10', '3.11'
```

Comment:
Will this work? Will it actually install 2 versions?

Reply:
sorry for that -- my `sed` went too wild, will fix now.